charts/gardener/controlplane/values.yaml

```
@@ -93,6 +93,10 @@ global:
         memory:
           value: "5G"
           percentage: 80
+    # List of admission plugins that should be disabled although they are in the default enabled plugins list.
+    disableAdmissionPlugins: []
```

Just to make sure these values are optional, can you comment them? 
```suggestion
  # disableAdmissionPlugins: []
```